### PR TITLE
Localization for TW server

### PR DESCRIPTION
**Pull request by the3dsandwich**
Just a quick question: is the option to switch languages present to test the translations?

Anyway, I think I have a (mostly) workable translation for TW server.

### CODE REVIEW

**Comment by MathewSachin** [comment]
There is no switch in the app for the language. If a translation is available for the language is set on phone, it is used by the app too. That's the way most Android apps work.

**Comment by the3dsandwich** [comment]
Hey @MathewSachin I had most of the translation done and ready. It should be good to merge.

There are a couple places where there's no localization text available to translate so it's still in English. I also left `Debug mode` in English so people will be a little bit more cautious before clicking it.

Silver x0
Screenshot_1599559720

Wait
Screenshot_1599559876

**Comment by MathewSachin** [comment]
I'm aware that the `Silver x0` is not translatable right now.

But, the `Wait` one I missed.
It should be fairly easy to add. Do you want to give it a try? I'm away from my development machine right now.
You'll just have to add a string in the English `localized.xml`, then use it in the Fine-tune screen, and put a corresponding translation.

**Comment by the3dsandwich** [comment]
Thanks for pointing me where I need to change! I found the setting and changed it along with the corresponding localization, now they should all be localized for TW.

Screenshot_1599561688

btw I wasn't really aware of (or using) the GamePress foru, maybe I can find something to help out by looking through it (making a video for the updated version seems nice...)

**Review by MathewSachin — APPROVED** [review]

**Comment by reconman** [comment]
@ScathachSkadi Can we add a KR translation of "Wait" as 딜레이?

**Comment by MathewSachin** [comment]
> btw I wasn't really aware of (nor using) the GamePress forum

yeah, you can find a lot of answers/info there.

> But maybe I can find something to help out by looking through it (making a video for the updated version seems doable...)

A video would really help us. I tried making one for AutoSkillMaker supporting edits, not sure if it's good enough for people to understand 🤣 
https://www.youtube.com/watch?v=TacAfvtCuPo

**Comment by reconman** [comment]
I can also try doing an updated video of the English version after I'm done with switching apartments. I have some subscribers waiting for new videos.

**Comment by ScathachSkadi** [comment]
> @ScathachSkadi Can we add a KR translation of "Wait" as 딜레이?

Yes, most of the time that translation will fit anywhere.

File: app/src/main/res/values-b+zh+TW/localized.xml [hunk 1]
```diff
@@ -1,2 +1,264 @@
...
+    <string name="p_fine_tune_stage_counter_similarity">關卡計數相似度 (%)</string>
+    <string name="p_fine_tune_clicks">點選</string>
+    <string name="p_fine_tune_wait_after_clicking">點選後暫停 (ms)</string>
+    <string name="p_fine_tune_click_duration">點選時間 (ms)</string>
+    <string name="p_fine_tune_click_delay">連擊間隔 (ms)</string>
+    <string name="p_fine_tune_swipes">滑動</string>
+    <string name="p_fine_tune_wait_after_swiping">滑動後暫停 (ms)</string>
+    <string name="p_fine_tune_swipe_duration">滑動時間 (ms)</string>
+    <string name="p_fine_tune_support_swipe_multiplier">支援選擇加乘 (%)</string>
+    <string name="p_fine_tune_wait">施放技能後等待時間</string>
```
Comment (reconman) [inline app/src/main/res/values-b+zh+TW/localized.xml:178]: This is not a translation of 'Wait', but something related to using skills. The multiplier affects ALL waiting times.

Please correct these translations.

Comment (the3dsandwich) [reply app/src/main/res/values-b+zh+TW/localized.xml:178]: Is `skill delay` the one that's related to using skills?
I personally wasn't sure what the Wait multiplier does.

Comment (reconman) [reply app/src/main/res/values-b+zh+TW/localized.xml:178]: Yes, only skill_delay

Comment (the3dsandwich) [reply app/src/main/res/values-b+zh+TW/localized.xml:178]: Cool. Then I think we can do something like
`wait` = `等待時間`
`skill_delay` = `施放技能後等待時間`?

File: app/src/main/res/values-b+zh+TW/localized.xml [hunk 2]
```diff
@@ -1,2 +1,264 @@
...
+    <string name="auto_skill_maker_atk_cards_before_np">寶具前墊刀</string>
+    <string name="auto_skill_maker_atk_done">完成</string>
+    <string name="auto_skill_maker_atk_next_wave">下一波</string>
+    <string name="auto_skill_maker_atk_next_turn">下一回合\n(同一波敵人)</string>
+
+    <string name="auto_skill_maker_main_enemy">敵人 %d</string>
+    <string name="auto_skill_maker_main_battle">BATTLE %d</string>
+    <string name="auto_skill_maker_main_turn">%d 回合</string>
+    <string name="auto_skill_maker_main_master_skills">御主技能</string>
+    <string name="auto_skill_maker_main_undo">回復</string>
```
Comment (reconman) [inline app/src/main/res/values-b+zh+TW/localized.xml:46]: Shouldn't this be 撤消? Doesn't 回復 mean "reply"?

File: app/src/main/res/values-b+zh+TW/localized.xml [hunk 3]
```diff
@@ -1,2 +1,264 @@
...
+    <string name="support_img_namer_file_rename_failed">重新命名成 \'%s\' 失敗</string>
+
+    <string name="p_script_mode">腳本模式</string>
+    <string name="p_refill">自動回體</string>
+    <string name="p_refill_off">關閉</string>
+    <string name="p_refill_summary">設定自動回體</string>
+    <string name="p_auto_skill">周回腳本</string>
+    <string name="p_auto_skill_summary">設定周回腳本</string>
+    <string name="p_more_options">更多</string>
+    <string name="p_update_available">更新</string>
```
Comment (reconman) [inline app/src/main/res/values-b+zh+TW/localized.xml:98]: Shouldn't this be something like 有可用的更新?

File: app/src/main/res/values-b+zh+TW/localized.xml (hunk 2 above)
Comment (the3dsandwich) [reply app/src/main/res/values-b+zh+TW/localized.xml:46]: weird. I was certain I saw it translated as 回復 or 復原 somewhere. Let me check on some of my apps...

File: app/src/main/res/values-b+zh+TW/localized.xml (hunk 3 above)
Comment (the3dsandwich) [reply app/src/main/res/values-b+zh+TW/localized.xml:98]: I'll fix this one. It's done when I was still trying not to be so wordy (I became wordy afterwards)

File: app/src/main/res/values-b+zh+TW/localized.xml (hunk 2 above)
Comment (the3dsandwich) [reply app/src/main/res/values-b+zh+TW/localized.xml:46]: (Screenshot from SnapSeed, an app by Google)
I suppose it's neither 撤銷 nor 回復. I'll change it to 復原.
Screenshot_20200908-233750

File: app/src/main/res/values-b+zh+TW/localized.xml [hunk 4]
```diff
@@ -1,2 +1,264 @@
...
+
+    <string name="p_advanced">進階</string>
+    <string name="p_debug_mode">Debug mode</string>
+    <string name="p_debug_mode_summary">Highlight the regions being matched</string>
+    <string name="p_ignore_notch">忽略瀏海</string>
+    <string name="p_ignore_notch_summary">對三星手機有幫助</string>
+    <string name="p_record_screen">螢幕錄影</string>
+    <string name="p_record_screen_summary">將儲存 \'record.mp4\' 至 \'Fate-Grand-Automata\' 資料夾，不適用root。</string>
+    <string name="p_root_screenshot">以root截圖</string>
+    <string name="p_root_screenshot_summary">對Android 7機種有幫助，請在開啟服務前選取此設定</string>
```
Comment (reconman) [inline app/src/main/res/values-b+zh+TW/localized.xml:157]: You forgot to include that it's only useful for Nox with Android 7. Nox is called 夜神 in Chinese.

File: app/src/main/res/values-b+zh+TW/localized.xml (hunk 1 above)
Comment (reconman) [reply app/src/main/res/values-b+zh+TW/localized.xml:178]: Seems fine, just add a (ms) after skill_delay, so `施放技能後等待時間 (ms)`

File: app/src/main/res/values-b+zh+TW/localized.xml (hunk 4 above)
Comment (the3dsandwich) [reply app/src/main/res/values-b+zh+TW/localized.xml:157]: I see. Did not notice that Nox is the name of the emulator. My bad.

**Comment by the3dsandwich** [comment]
@reconman Is there a way to add to this PR? Or should I create another PR like I just did. I don't want to look like I'm spamming PRs

**Comment by reconman** [comment]
Another PR, there's no way to reopen this one.

**Pull request merged**